pom.xml

```diff
@@ -118,18 +118,18 @@
     </dependency>
     <dependency>
       <groupId>com.google.http-client</groupId>
-      <artifactId>google-http-client-jackson2</artifactId>
-      <version>1.39.2</version>
+      <artifactId>google-http-client-jackson</artifactId>
+      <version>1.29.2</version>
```

Comment:
Why the version downgrade? Maybe worth a code comment.

Reply:
We went from jackson to jackson 2 recently and I realized that when I was running it in helltool (even master) it was generating an error 🤷‍♀️

Reply:
What kind of an error?  I did that upgrade and I noticed that we needed an accompanying change on the Looker side, but I added that and it seemed to be working.

Reply:
definitely down for it then

Reply:
What are the helltool changes you needed to make ?

Reply:
https://github.com/looker/helltool/pull/64605 (already merged)